### ISSUE TITLE
Drop down menus faulty/buggy

### ISSUE THREAD

**Issue by SoilnRock** [Manuel-777/MTG-Arena-Tool#1213]
There's kind of  a refresh-error with some drop down menus, especially in collection tab. Most obvious when you try to select the same entry the menu showed before.

**Comment by riQQ** [comment]
Could you explain in more detail how to reproduce the issue?

**Comment by SoilnRock** [comment]
The drop down menus in collection tab: Suppose it is set to "In boosters", you click on it and for whatever reason you want to select "In boosters" again - which doesn't work. Also sometimes the selected entry doesn't mactch what is actually shown in the collection, for example sometimes it is set to "All cards" but only cards "not in boosters" are shown.
I think Manwe knows about this he said something about having repaired this menus somewhere else already I believe.

**Comment by riQQ** [comment]
I can reproduce
- clicking on the currently selected entry doesn't close the expanded drop-down
- clicking outside of the expanded drop-down doesn't close it

I haven't yet encountered the issue with the drop-down showing the wrong selected entry.

**Comment by SoilnRock** [comment]
Yeah, the wrong showing occurs rather seldomly, haven't found a way to reproduce it, seems a little random.

**Comment by SoilnRock** [comment]
<img width="1457" alt="jjjj" src="https://user-images.githubusercontent.com/32754919/104188246-3aa8f900-5419-11eb-81a4-8db9e5078b9a.png">
Here you can see the wrong showing "in action". It's obvious that for whatever reason "not in bossters" is active instead of "all cards" as shown in the drop down menu.

**Comment by guru-of-sound** [comment]
There is also an issue with the format drop down menu within the Cards tab.
